src/requests/team.ts

```diff
@@ -1,14 +1,14 @@
 import type { GraphQLClient } from "graphql-request";
 import { gql } from "graphql-request";
 
-import { TeamRole } from "../types/roles";
+import { Role } from "../types/roles";
 import { Team, TeamSettings, TeamTheme } from "../types/team";
 import { decrypt } from "../utils/encryption";
 
 interface UpsertMember {
   userId: number;
   teamId: number;
-  role: TeamRole;
+  role: Role;
```

Comment:
I think we might actually want this to be `role: TeamRole | UserRole` which would still get you `platformAdmin`, while restricting/omitting `HasuraRole` (we never want to upsert a member with role "public" !) ? 

Does that make sense? 

Trying to think of correct name for that grouping of roles so could re-use below too, but coming up blank - `TeamRoleWithAdmin` ?? :sweat_smile:

Reply:
Ah yes that totally makes sense!

Naming wise, maybe something in opposition to `HasuraRole`....`EditorRole`?

Reply:
Perfect - I like `EditorRole` :+1: